db/tx.go

```diff
@@ -40,3 +41,12 @@ func RunMulti(tx *sqlx.Tx, fns []TxFunc) error {
 	}
 	return nil
 }
+
+func RunMultiExt(tx sqlx.Ext, fns []ExtFunc) error {
```

Comment:
Parameter naming should be accurate. Tx implies transaction when you are using ext.